SSH/legos/ssh_scp/ssh_scp.py

```diff
@@ -51,31 +56,16 @@ def ssh_scp(
         :type local_file: str
         :param local_file: Filename on the unSkript proxy. Eg /tmp/my_local_file
 
+        :type proxy_host: str
+        :param proxy_host: Proxy Host to connect host via. Eg 10.10.10.10.
+
         :type direction: bool
         :param direction: Direction of the copy operation. Default is receive-from-remote-server
 
         :rtype:
     """
-    client = sshClient([host])
+    client = sshClient([host], proxy_host)
     copy_args = [{'local_file': local_file, 'remote_file': remote_file}]
-
-    if direction is True:
-        cmds = client.copy_remote_file(remote_file=remote_file, local_file=local_file,
-                                       recurse=False,
-                                       suffix_separator="", copy_args=copy_args,
-                                       encoding='utf-8')
-
-    else:
-        cmds = client.copy_file(local_file=local_file, remote_file=remote_file,
-                                recurse=False, copy_args=None)
-
-    try:
-        joinall(cmds, raise_error=True)
-        if direction is True:
-            return f"Successfully copied file {host}://{remote_file} to {local_file}"
-        else:
-            return f"Successfully copied file {local_file} to {host}://{remote_file}"
-
-    except Exception as e:
-        return f"Error encountered while copying files {e}"
+    cmds = client.copy_file(local_file, remote_file, direction)
```

Comment:
do we have copy_file ?  i read somewhere the suggested way was

client = sshClient([host], proxy_host)
scp = client.openSFTP()
scp.put(....) ?